lib/engines/mongodb.js

```diff
@@ -67,31 +67,36 @@ MongoCollection.prototype.findOne = function(query, projection, callback) {
 };
 MongoCollection.prototype.find = function(query, projection) {
 	objectifyID( query );
+	console.log( JSON.stringify( query ) );
 	return new MongoCursor( this.db.find(query, projection) );
 };
 MongoCollection.prototype.insert = function(obj, opt, callback) {
-	return this.db.insertAsync(obj, opt)
-		.then(function(arr) {
-			_.forEach(arr, function(item) {
+	return this.db.insertOneAsync(obj, opt)
+		.then(function(res) {
+			if ( res.result.ok != 1 )
```

Comment:
What's our "if/then" style? Feels like open brace should be on the same line as the condition. /nitpick